tools/lib/provision.py

```diff
@@ -262,7 +264,27 @@ def main(options):
         else:
             print("No need to regenerate the test DB.")
 
-        run(["./manage.py", "compilemessages"])
+        # Check if translation files have been modified or not
+        sha1sum = hashlib.sha1()
+        paths = glob.glob('static/locale/*/LC_MESSAGES/*.po')
+        paths += glob.glob('static/locale/*/translations.json')
+
+        from django.utils.encoding import force_bytes
```

Comment:
we should avoid importing Django; doing so will add like 0.8s to the runtime.  `zerver.lib.str_utils.force_bytes` is a better version of that function to use.